app/build.gradle

```diff
@@ -93,11 +93,11 @@ android {
         buildConfigField "String", "PERMISSION_LOCAL_BROADCAST", "\"${localBroadcastPermission}\""
     }
 
-    // testOptions {
-    //     unitTests.all {
-    //         useJUnitPlatform()
-    //     }
-    // }
+    testOptions {
```

Comment:
See this change which breaks exactly what you are experiencing @mahibi hence I killed it but this revert brings it back since this has been merged to master via offline capabilities PR by @rapterjet2004